tests/functional/TestSmokeTest/vagrant/roles/common/tasks/main.yml

```diff
@@ -10,10 +10,11 @@
     - /dev/vdh
     - /dev/vdi
 
-- name: install packages
+- name: install repo source packages
   yum: name={{ item }} state=present
   with_items:
     - epel-release
+    - centos-release-storage-common
     - centos-release-gluster
```

Comment:
Please try with `centos-release-gluster41` so that [Bug 1659563](https://bugzilla.redhat.com/show_bug.cgi?id=1659563) that was introduced with gluster-5 is not hit.